src/playground/service.clj

```diff
@@ -32,7 +33,10 @@
   (ring-resp/response (views/home request)))
 
 (defn insert-page [request]
-  (ring-resp/response (views/insert request)))
+  (if (authenticated? (:session request))
```

Comment:
Instead of an `if`, implement an authorization interceptor. It should terminate the request with HTTP 403 if the role isn't the expected one.

Reply:
done

Reply:
I think you forgot to remove the `if`

Reply:
I removed it in the `delete`, the `insert` on the other hand just requires a session, not an admin role. Which is what I was expecting the `authentication-interceptor` and backend to solve. I'll look further and get back to you.

Reply:
implemented.